pay/lib/src/pay.dart

```diff
@@ -34,12 +34,20 @@ class Pay {
   Pay(this._configurations) : _payPlatform = PayMethodChannel();
 
   /// Alternative constructor to create a [Pay] object with a list of
-  /// configurations in [String] format.
+  /// configuration assets in [String] format.
   Pay.withAssets(List<String> configAssets)
       : _payPlatform = PayMethodChannel() {
     _assetInitializationFuture = _loadConfigAssets(configAssets);
   }
 
+  /// Alternative constructor to create a [Pay] object with a list of
+  /// configurations in [String] format.
+  Pay.withStrings(List<String> configStrings)
+      : _payPlatform = PayMethodChannel() {
+    _assetInitializationFuture = Future.value(
```

Comment:
This does not seem to set the `_configurations` prop that is later used to configure the payment providers.
Since loading configs from a string is a synchronous task, it can be included as part of the initializer list directly. For example:
```dart
Pay.withStrings(List<String> configStrings)
      : _payPlatform = PayMethodChannel(),
        _configurations = configStrings
            .map((cs) => PaymentConfiguration.fromJsonString(cs))
            .toList();
```